app/main/controllers/template/rtmedia-functions.php

```diff
@@ -692,9 +692,9 @@ function rtmedia_image( $size = 'rt_media_thumbnail', $id = false, $recho = true
 	$src = apply_filters( 'rtmedia_media_thumb', $src, $media_object->id, $media_object->media_type );
 
 	if ( true === $recho ) {
-		echo set_url_scheme( $src );
+		echo set_url_scheme( $src ) . '?' . time();
```

Comment:
@kanumalivad Please add comment why the timestamp was added, for future references.

Reply:
okk